src/js/providers/flash.js

```diff
@@ -338,6 +341,10 @@ define([
                         this.trigger(events.JWPLAYER_MEDIA_ERROR, event);
                     }, this);
 
+                    _swf.on('mediaType', function(e) {
+                        this.trigger('mediaType', {mediaType: e.mediaType});
```

Comment:
You should be able to just forward the event object `e` here, instead of creating a new object and copying the property we're looking for off it.